site/ja/lite/performance/post_training_quantization.md

```diff
@@ -0,0 +1,164 @@
+# 訓練後の量子化
+
+訓練後の量子化は、わずかなモデルの精度の低下を伴いますが、
+モデルの大きさを削減することができ、さらにCPUとハードウェアアクセラレータのレイテンシーを改善する変換手法です。
+[TensorFlow Lite Converter](../convert/)
+を使って TensorFlow Lite 形式に変換するときに、訓練済みの浮動小数点数の TensorFlow モデルを使ってこれらの手法を実行できます。
+
+注意: このページの関数はTensorFlow 1.15 以上が必要です。
```

Comment:
[strong suggestion]
```suggestion
Note: このページの関数はTensorFlow 1.15 以上が必要です。
```

In the following comment, I described the reason why we should leave "Note:" untranslated. (Sorry, I found this line after writing the comment describing it.)

Reply:
i used your suggestion.